types/v56/types.go

```diff
@@ -373,6 +373,49 @@ type Vdc struct {
 	VMQuota            int                   `xml:"VmQuota"`
 }
 
+// VdcStorageProfile represents the parameters to create a storage profile in an organization vDC.
+// Type: VdcStorageProfileParamsType
+// Namespace: http://www.vmware.com/vcloud/v1.5
+// Description: Represents the parameters to create a storage profile in an organization vDC.
+// Since: 5.1
+// https://code.vmware.com/apis/220/vcloud#/doc/doc/types/VdcStorageProfileParamsType.html
+type VdcStorageProfile struct {
```

Comment:
I agree it isn't obvious, but isn't that just a result of the API definitions?  The API defines [VdcStorageProfilesType](https://code.vmware.com/apis/220/vcloud#/doc/doc/types/VdcStorageProfilesType.html) and [VdcStorageProfileType](https://code.vmware.com/apis/220/vcloud#/doc/doc/types/VdcStorageProfileType.html) differently, so it seems that design is just being followed here.  I don't mind deviating if you think it's best; maybe we could call this `VdcStorageProfileDefinition` or something since it actually has the data elements (rather than just being a reference).

Reply:
Also perhaps I'm in the wrong mindset of trying to follow the API too closely.  I see one of your other comments for changing `CreateVdcParams` to `VdcConfiguration`, so it seems that's not a goal here?